packages/shared/src/styles/components/buttons.css

```
@@ -176,6 +176,7 @@
 
   & :hover ~ label, & .hover ~ label, & :focus:focus-visible ~ label {
     color: var(--button-hover-color);
+    transition: all 0.2s ease-in-out;
```

Same goes here, just tagging for visibility 🚢 
```suggestion
    transition: background-color 0.2s ease-in-out;
```